app/services/passporting/ioj_passporter.rb

```diff
@@ -0,0 +1,28 @@
+module Passporting
+  class IojPassporter < BasePassporter
+    def call
+      ioj_passport = []
+      ioj_passport << IojPassportType::ON_AGE_UNDER18 if applicant_under18?
+
+      crime_application.update(ioj_passport:)
+
+      passported?
+    end
+
+    def passported?
+      # IoJ passporting can be overridden for applications returned
+      # back to the provider due to the case being split
```

Comment:
Caseworkers shouldn't return a split application for an under 18 applicant, but just wondering if the logic works here if they do. The under 18 applicant should still be passported (and also that calculation should be from their age on application submission, not re running the passporter at time of resubmission)

Reply:
Yes, the logic works whether the IoJ passporting was due to under 18 (in fact this is the only one that we can trigger on staging) or, soon, due to offence passporting.

More granular behaviour or edge cases like the one you mention, will be best done as separate PR(s), as this one is already quite large unfortunately.

I think the most likely solution is that the age check will be run only on new applications and not on resubmissions.